### PR TITLE
Adding resource reboot logic

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a127472](https://github.com/bobbytreed/PowerShell-Docs/pull/3556/commits/a1274723f8e6f1749e2c93f327d599deb3b5562b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a127472](https://github.com/bobbytreed/PowerShell-Docs/pull/3556/commits/a1274723f8e6f1749e2c93f327d599deb3b5562b)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 19 sec(s)_

File: dsc/resources/authoringResourceMOF.md [hunk 1]
```diff
@@ -287,3 +287,16 @@ if (PsDscContext.RunAsUser) {
...
 ```
+
+## Rebooting the Node
+
+If the actions taken in your `Set-TargetResource` function require a reboot, you can use a global flag to tell the LCM to reboot the Node. This reboot occurs directly after the `Set-TargetResource` function completes.
+
+Inside your `Set-TargetResource` function, add the following line of code.
+
+```powershell
+Include this line if the resource requires a system reboot.
```
Comment (sdwheeler) [inline dsc/resources/authoringResourceMOF.md]: Is this line supposed to be a comment or instructions? Comment it out or move it outside of the code block.

Comment (bobbytreed) [reply dsc/resources/authoringResourceMOF.md]: Good catch, simple typo.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3e8bdb4](https://github.com/bobbytreed/PowerShell-Docs/pull/3556/commits/3e8bdb4b717dca48b1a7480b31819849b68dd513)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3e8bdb4](https://github.com/bobbytreed/PowerShell-Docs/pull/3556/commits/3e8bdb4b717dca48b1a7480b31819849b68dd513)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 19 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3e8bdb4](https://github.com/bobbytreed/PowerShell-Docs/pull/3556/commits/3e8bdb4b717dca48b1a7480b31819849b68dd513)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/resources/authoringResourceMOF.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1400683/dsc/resources/authoringResourceMOF.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/resources/authoringresourcemof?branch=pr-en-us-3556) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C16%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901161545034802-3556%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NhKugDwT91K8dVqdyMX%2FU2Q0zLSgHrkqXSpuWE9l2E4%3D&st=2019-01-16T15%3A54%3A26Z&se=2019-02-16T15%3A59%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**